README.md

```diff
@@ -52,6 +52,18 @@ to document the current shortcomings of this tool.
 
 * Go plugins are not currently supported; see [#87](https://github.com/burrowers/garble/issues/87).
 
+* There are cases where garble is a little to agressive with obfuscation, this may lead to identifiers getting obfuscated which are needed for reflection, e.g. to parse JSON into a struct; see [#162](https://github.com/burrowers/garble/issues/162). To work around this you can hint garble, that an type is used for reflection via passing it to `reflect.TypeOf` or `reflect.ValueOf` in the same file:
```

Comment:
Maybe instead of 'To work around this you can hint garble' say 'To work around this you can pass a hint to garble'